docker/zap-api-scan.py

```diff
@@ -228,14 +238,14 @@ def main(argv):
     if len(target) == 0:
         usage()
         sys.exit(3)
-    if format != 'openapi' and format != 'soap':
-        logging.warning('Format must be either \'openapi\' or \'soap\'')
+    if format != 'openapi' and format != 'soap' and format != 'graphql':
+        logging.warning('Format must be either \'openapi\', \'soap\', or \'graphql\'')
         usage()
         sys.exit(3)
 
     if running_in_docker():
         base_dir = '/zap/wrk/'
-        if config_file or generate or report_html or report_xml or report_json or report_md or progress_file or context_file or target_file:
+        if config_file or generate or report_html or report_xml or report_json or report_md or progress_file or context_file or target_file or schema_file:
```

Comment:
"target_file" and "schema_file" will always be empty at this point in the program. Should they be removed?

Reply:
Just the fact that they have been specified means the user will need to mount a local dir, so this is good.

Reply:
They are not being specified though, they are initialised with the actual values later. Not sure if best Python practice but declaring them later would be more clear.